test/dataset_test.cpp

```diff
@@ -1535,15 +1536,15 @@ TEST(Dataset, counts_toDensity_fromDensity) {
   Dataset d;
   d.insert(Coord::X, {Dim::X, 4}, {1, 2, 4, 8});
   d.insert(Data::Value, "", {Dim::X, 3}, {12, 12, 12});
-  d(Data::Value, "").setUnit(Unit::Id::Counts);
+  d(Data::Value, "").setUnit(units::counts);
 
   d = counts::toDensity(std::move(d), Dim::X);
   auto result = d(Data::Value, "");
-  EXPECT_EQ(result.unit(), Unit::Id::CountsPerMeter);
+  EXPECT_EQ(result.unit(), units::counts / units::m);
```

Comment:
As mentioned in `units.h` when declaring the allowed set of units, `counts / m` is a little strange here.
What about `counts / us` for when we have a distribution of counts divided by bin width?